vscode-extension/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "relay-lsp-client",
+  "version": "1.0.0",
+  "description": "Relay LSP Client for VSCode",
+  "main": "./out/extension.js",
+  "activationEvents": [
+    "onLanguage:plaintext",
+    "onLanguage:javascript",
+    "onLanguage:javascriptreact",
+    "onLanguage:typescript",
+    "onLanguage:typescriptreact"
+  ],
+  "contributes": {
+    "configuration": {
+      "type": "object",
+      "title": "Relay",
+      "properties": {
+        "relay.enabled": {
+          "scope": "window",
+          "type": "boolean",
+          "default": true,
+          "description": "Is relay enabled"
+        },
+        "relay.outputLevel": {
+          "scope": "window",
+          "type": "string",
+          "default": "quiet-with-errors",
+          "description": "Can be one of the following. `quiet`, `quiet-with-errors`, `verbose`, `debug`"
```

```suggestion
          "description": "Controls what is logged to the Output Channel."
```

Maybe we can use the enum/enum description as outlined here? https://code.visualstudio.com/api/references/contribution-points#Configuration-property-schema